vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -207,10 +212,11 @@ def refresh_infrastructure_cache(self):
         self.log.debug("Refreshing the infrastructure cache...")
         t0 = Timer()
         infrastructure_data = self.api.get_infrastructure()
+        collect_property_metrics = self._config.collect_property_metrics
         self.gauge(
             "datadog.vsphere.refresh_infrastructure_cache.time",
             t0.total(),
-            tags=self._config.base_tags,
+            tags=self._config.base_tags + ['collect_property_metrics:{}'.format(collect_property_metrics)],
```

Comment:
What is the purpose of this new tag?

Reply:
It adds another level of debugging- In the case the time to refresh the cache increases dramatically when enabling this option, we'll be able to tell right when the flag has been enabled/disabled and see how much it correlates with the refresh time.